frontend/src/pages/User/Login/Login.js

```diff
@@ -0,0 +1,131 @@
+import {
+  Typography,
+  Box,
+  Grid,
+  Link,
+  TextField,
+  CssBaseline,
+  Button,
+  Avatar,
+  Paper,
+} from "@mui/material";
+import { Link as RouterLink } from "react-router-dom";
+import { School } from "@mui/icons-material";
+import { LeftGrid } from "./Styles";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { schema } from "./FormSchema";
+
+export default function SignInSide() {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    // clearErrors,
+    // setError,
```

Comment:
```suggestion
```
Should not maintain commented lines like this.
If needed, you put a comment for transparency. Below is a sample comment
`// TODO: To be used in integration`